tests/components/google_generative_ai_conversation/test_init.py

```diff
@@ -211,22 +220,15 @@ async def test_generate_content_service_with_non_image(hass: HomeAssistant) -> N
     ("side_effect", "state", "reauth"),
     [
         (
-            ClientError("some error"),
+            CLIENT_ERROR_500,
             ConfigEntryState.SETUP_ERROR,
             False,
         ),
         (
-            DeadlineExceeded("deadline exceeded"),
+            Timeout,
             ConfigEntryState.SETUP_RETRY,
             False,
         ),
-        (
```

Comment:
Can you add this test case back? It seems Paulus removed it earlier to make it pass